packages/cms-base-layer/app/components/SwQuantitySelect.vue

```diff
@@ -8,13 +34,22 @@ const {
   steps,
   max,
   min,
+  id: propId,
 } = defineProps<{
   size?: "small" | "large";
   steps?: number;
   min?: number;
   max?: number;
+  id?: string;
 }>();
 
+// generate an id that prefers a provided prop and otherwise uses the component uid
+const inputId = computed(() => {
+  if (propId) return propId;
+  const uid = getCurrentInstance()?.uid ?? Math.floor(Math.random() * 1e6);
```

Comment:
https://github.com/vuejs/vue/blob/main/src/v3/currentInstance.ts


```suggestion
  const uid = Math.random().toString(36).substr(2, 9);
```